README.md

```diff
@@ -52,6 +52,7 @@ $ pip3 install kas
 
 Now you can build the example image like this:
 
+**_Warning:_** Make sure that the working directory (meta-iot2000) is located in a EXT4 partion.   
```

Comment:
I would relax this toward "[...] is located on a native Linux file system such as ext4, xfs, btrfs etc. NTFS is known to **not** work." And please place the comment before "Native Build" because it applies to both methods as you noticed.

Reply:
Thank you for the help in this first PR.